Fabio polish2023 11 (#13)
open-make/airtrack-doc-website: basicframework.md

In this set we will attach the frame (blue marked in the picture above)  to the air table box.

> - Place four [nut](connectors.yml#5mmNuts){Qty:8}s inside each 480 mm rod of the U-formation frameworks on one side which adjacent to the side with nuts already placed in (90 degrees to both rods which can vary in length), both sides work. 

Author: 4 or 8? explain

Reviewer: it is two times 4

Author: ok

![](images/lower_frame.jpg)

- Install [angle](connectors.yml#5mmNuts_angle){Qty:8} with a [M5 screw (6mm)](screws.yml#m5x6mm_screw){Qty:8} and 
[nut](connectors.yml#5mmNuts){Qty:8}s on the ends of two [aluminium strut profiles (length: 440mm)](fromstep){Qty:2} and two [aluminium strut profiles (length: 286mm)](fromstep){Qty:2}.

- Put an extra [nut](connectors.yml#5mmNuts){Qty:2} in both 440 mm aluminium strut profiles. It must point to the center later (90° shifted from the other two nuts). This will be used to add an horizontal strut later on.

- Connects these stuts with the table feets using [nut](connectors.yml#5mmNuts){Qty:8} and [M5 screw (6mm)](screws.yml#m5x6mm_screw){Qty:8}, placed at the wished height (10 mm works). The angle should be place below the horizontal stuts.

>i **Note** 
>i

> >i The following strut is only needed if you want to place the camera on it, it has no stability reasons

Author: ```suggestion
>i The following strut is only needed if you want to place the camera on it, it does not have a stability purpose
```

Reviewer: suits
